### DISCUSSION TITLE
Windows 11 taskbar resized icons no longer vertically centered after reboot

### DISCUSSION THREAD

**Discussion by laptop333567** [ramensoftware/windhawk-mods#547] [Q&A] [answered]
Hi, 

I have tried searching the forum and wiki for this question, but so far haven't been able to find any information on it.

I'm new to both Windows 11 and Windhawk, which I've downloaded in order to make the taskbar and its icons smaller. I am using the Mods "Taskbar height and icon size"  and " Taskbar Clock Customization". After saving the adjusted setting of the first Mod the taskbar looks sooo much better (thank you!!) and the smaller icons are nicely aligned vertically (ie, positioned in the middle of the taskbar height wise).

However, after I've rebooted Windows, the taskbar is still "thinner" and the icons smaller, but the icons have dropped somewhat on the taskbar height wise. I have to open Windhawk, adjust a setting in the taskbar mod (simply so that I can press on save), and then the vertical alignment looks okay again. Until the next reboot...

Is this a bug? The workaround is doable, since I prefer not to reboot often anyway, but sometimes it's unavoidable.

I have made and will try to attach screenshots.

How the icons look after saving the new settings: 
![icons OK](https://github.com/ramensoftware/windhawk/assets/161061626/ad3570eb-6fd1-4bf3-85b3-da1f2a1c430d)

How the icons look after the reboot (this is the lower left corner of my screen, so I haven't cut anything off or neglected to screenshot it):
![icons dropped](https://github.com/ramensoftware/windhawk/assets/161061626/2a42be25-f12e-47f9-b18a-931d01e857ba)

Regards, 
Suzanne

**Comment by laptop333567** [comment]
Sorry, having some difficulty with the screenshots... this is the last one, my Windhawk settings:
![windhawk settings](https://github.com/ramensoftware/windhawk/assets/161061626/1493d2ec-8e03-47cb-8065-ecaf7c1176d6)

Regards, 
Suzanne

**Comment by m417z** [comment, marked as answer]
Looks similar to this issue: https://github.com/ramensoftware/windhawk-mods/issues/529
I wasn't able to reproduce the issue, but I made some improvements and it might be fixed in Taskbar height and icon size v1.2.3. Please try it and let me know whether it's fixed for you.

**Comment by laptop333567** [comment]
Hi, 

My issue has been fixed! Thank you so much! 

Just as a FYI: I just updated Win11 (Home) and the Windhawk taskbar mod (I'm now on version 1.2.4), rebooted my laptop, adjusted some of the mod's settings, rebooted again, and the taskbar still looks just the way I want it to.  :)

Re. issue 529: 
It does sound similar, but my laptop does not have a touch (screen?) mode. I hadn't tried restarting Windows Explorer for the issue I had, since I just went straight to the Windhawk mod after rebooting and noticing the shifted icons, and (re)adjusted the settings there. 

Re. issue 413:
I didn't have a problem with full screen windows. I don't hide my taskbar. Screen resolution is set to 1920x1080.

Regards,
Suzanne

**Comment by laptop333567** [comment]
The person who posted issue 413 should probably test if it is fixed for them, but I just hid my taskbar, just for the heck of it; no problem with full screens. Rebooted my laptop; still no problem. My taskbar mod is currently set to: 20, 35, 40.

Regards,
Suzanne
